src/app/modules/browse/browse-controller.js

```diff
@@ -4,6 +4,7 @@
 	.config(['$stateProvider', function config($stateProvider) {
 
 		$stateProvider.state('browse', {
+            parent: 'requireLoginResolution',
```

Comment:
That is strange. On WebStorm, I don't see these kinds of inconsistencies. They all look the same. Generally speaking, I don't do any indentation; WebStorm automatically does the indentation it wants to do. WebStorm seems to prefer spaces.